app/src/ui/lib/sandboxed-markdown.tsx

```diff
@@ -88,22 +103,50 @@ export class SandboxedMarkdown extends React.PureComponent<
    * We still want to be able to navigate to links provided in the markdown.
    * However, we want to intercept them an verify they are valid links first.
    */
-  private setupLinkInterceptor(frameRef: HTMLIFrameElement): void {
+  private setupFrameLoadListeners(frameRef: HTMLIFrameElement): void {
     frameRef.addEventListener('load', () => {
-      frameRef.contentDocument?.addEventListener('click', ev => {
-        const { contentWindow } = frameRef
+      this.setupLinkInterceptor(frameRef)
+      this.setupFrameScrollHeightListener(frameRef)
+    })
+  }
 
-        if (contentWindow && ev.target instanceof contentWindow.Element) {
-          const a = ev.target.closest('a')
-          if (a !== null) {
-            ev.preventDefault()
+  /**
+   * Iframes without much styling help will act like a block element that has a
+   * predetermiend height and width and scrolling. We want our iframe to feel a
+   * bit more like a div. Thus, we want to capture the scroll height, and set
+   * the container div to that height and with some additional css we can
+   * achieve a inline feel.
+   */
+  private setupFrameScrollHeightListener(frameRef: HTMLIFrameElement): void {
```

Comment:
Is this actually a listener? 🤔  Seems like it only changes the div height once, right?

Reply:
technically anytime the iframe loads..

Reply:
Which hopefully is only ones.. but it happens anytime the iframe onload event fires.

Reply:
Actually... that may fire again if you changed the markdown .. which we would want to happen

Reply:
😜  Tho I think your point that the method name is misleading just flew over my head. I renamed it.

Reply:
> 😜 Tho I think your point that the method name is misleading just flew over my head. I renamed it.

Yes 😂  Either it was misleading or I was missing something 😳